src/hotspot/share/memory/filemap.cpp

```diff
@@ -198,18 +198,18 @@ FileMapInfo::~FileMapInfo() {
   }
 }
 
-void FileMapInfo::populate_header(size_t alignment) {
-  header()->populate(this, alignment);
+void FileMapInfo::populate_header(size_t region_alignment) {
+  header()->populate(this, region_alignment);
 }
 
-void FileMapHeader::populate(FileMapInfo* mapinfo, size_t alignment) {
+void FileMapHeader::populate(FileMapInfo* mapinfo, size_t region_alignment) {
   if (DynamicDumpSharedSpaces) {
     _magic = CDS_DYNAMIC_ARCHIVE_MAGIC;
   } else {
     _magic = CDS_ARCHIVE_MAGIC;
   }
   _version = CURRENT_CDS_ARCHIVE_VERSION;
-  _alignment = alignment;
+  _region_alignment = region_alignment;
```

Comment:
I think this should be renamed to `_core_region_alignment` to be consistent with `MetaspaceShared::core_region_alignment()`